### ISSUE TITLE
Force player names to be server-sided without forcing their teams?

### ISSUE THREAD

**Issue by smpcheathunt** [shobhit-pathak/MatchZy#254]
Is it possible to have server-sided player names (avatars aswell), without being forced to their respective teams? Like in the match config, player can join any team they want but the names are server-sided as set in the configs.

**Comment by borzaka** [comment]
Yes, you can use the `sv_load_forced_client_names_file namesfile.txt` command.
Example file content:
```
“Names”
{
“765611979849XXXX” “Official Player Name”
“765611979800YYYY” “Other Player Name”
… etc.
}
```
